llpc/test/shaderdb/relocatable_shaders/PipelineVsFs_GlueCache.pipe

```diff
@@ -0,0 +1,121 @@
+; This test checks that all of the shaders in a graphics pipeline are correctly add to the shader cache and retrieved from it.
+; We only check the internal shader cache because of the limitation on the testing infrastructure.
+
+; BEGIN_SHADERTEST
+; RUN: rm -rf %t_dir && \
+; RUN: mkdir -p %t_dir && \
+; RUN: amdllpc -spvgen-dir=%spvgendir% %gfxip \
+; RUN:         -shader-cache-mode=2 \
+; RUN:         -shader-cache-filename=cache.bin -shader-cache-file-dir=%t_dir \
+; RUN:         -enable-relocatable-shader-elf \
+; RUN:         -o %t.elf %s -v | FileCheck -check-prefix=CREATE %s
+; REQUIRES: llpc-shader-cache
+; CREATE: Building pipeline with relocatable shader elf.
+; CREATE: Cache miss for shader stage vertex
+; CREATE: Updating the cache for shader stage vertex
+; CREATE: Cache miss for shader stage fragment
+; CREATE: Updating the cache for shader stage fragment
+; CREATE: ID for glue shader0: 00000000000000007632663332570000000200000003000000040000000F00000000000000030000000400000000000000000000000000000000000000000000000E0000000700000000000000
```

Comment:
Would it be possible print the IDs in some other format so that they don't end up as a very long number? I find it very hard to read, tell where each word/byte begins, and the check line itself wraps past the usual column limit

Reply:
I personally like a single string without spaces.  When debugging, and it is easy to just search for the "word" in vim or other editors.  You just put your cursor on it and hit `*`.  If it is divided into multiple words and split over multiple lines, then searching and comparing becomes harder.  So even if this is harder for a human to parse, with tooling it is easier to deal with.

Reply:
I don't see how hopping to the next space or `-` is more difficult, you can just do `f<CHAR>` and then repeat with `;`. But I'm fine with the current format if you like it more.